src/models/tag.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable strict */
+
+'use strict';
+
+const { Model, ValidationError } = require('objection');
+
+const tableTag = 'tag';
+class Tag extends Model {
+  static get tableName() {
+    return tableTag;
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['name'],
+      properties: {
+        type: { type: 'string', maxLength: 25 },
```

Comment:
type? is this supposed to be 'name' ?
this is probably why name's maxlength validation is not triggered

Reply:
Done